facebook/react-native
# [Trivial] Use Markdown extension in SUPPORT file

## Summary

Followup to fe80a0c01d6ef2ce0c2aebecad1474debce34e18. Use a Markdown extension to let GitHub know this can be rendered as a Preview, otherwise GitHub will display the plaintext version.




## Changelog



[General] [Fixed] - Use Markdown in SUPPORT file

## Test Plan

Verified on a test repo: 

1. Visit https://github.com/hramos/dotgithub/issues/new
2. Click the Support link on the sidebar
3. Observe https://github.com/hramos/dotgithub/blob/master/.github/SUPPORT.md is rendered to HTML.
